NARFO_BE/NARFO_BE/Controllers/MembersController.cs

```diff
@@ -2,121 +2,133 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Cors;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using NARFO_BE.Models;
 
 namespace NARFO_BE.Controllers
 {
-    [Route("api/[controller]")]
-    [ApiController]
+    [Route("Member")]
+    [EnableCors("MyPolicy")]
```

Comment:
I suggest you EnableCors for your whole project on startup.cs , it will save you from adding `[EnableCors("MyPolicy")]` on every controller

Except if you have other controllers that you want to restrict CORS for

Reply:
its already done.. check startup.cs

Reply:
Soo there is no need to have it on the individual controllers

Reply:
yes.. we'll remove it thank you for your view ( :

Reply:
personal preference...

Reply:
I'm not too sure about personal preferences here, using api/ as a prefix to endpoints is a REST best practice. Using plural nouns is also best practice, so `api/members` instead of `/member`

Reply:
Specifically, you'll see one of two things for APIs:
- if you have a wildcard namespace for the server, you may use `api.server.com/items`
- if you _don't_ have a wildcard namespace, you're forced to use `server.com/api/items`

You will always see one of the two, never `server.com/items`.

Reply:
ok sorted